Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM opensciencegrid/software-base:fresh
+
+# Install dependencies (application, Apache)
+RUN \
+    yum update -y && \
+    yum install -y \
+      gcc \
+      git \
+      libyaml-devel \
+      mod_ssl \
+      python3-devel \
+      python3-ldap \
+      python3-pip \
+    yum install -y \
+      fetch-crl \
+      httpd \
+      gridsite \
+      osg-ca-certs \
+      python3-mod_wsgi \
+    && yum clean all && rm -rf /var/cache/yum/*
+
+WORKDIR /app
+
+# Install application dependencies
+COPY requirements-rootless.txt src/ ./
+RUN pip3 install --no-cache-dir -r requirements-rootless.txt
```

Comment:
```suggestion
RUN pip3 install --no-cache-dir -r requirements-apache.txt
```

Reply:
although it looks like you're already installing mod_wsgi and python3-ldap so I could be wrong about this. 
You should check the `/stashcache/authfile` endpoint to make sure the results contain a bunch of lines like
```
u 47146b14.0 /user/ligo rl
```

Reply:
LDAP did seem to be working with the RPM version. I'll switch over to `requirements-apache.txt` and pip3 for installing the ldap and mod-wsgi modules. May as well keep the requirements together where possible, I guess!